convert/changeset.go

```diff
@@ -0,0 +1,16 @@
+package convert
```

Comment:
The `Change` struct should not be part of the `convert` package as this will be outsourced soon.

Reply:
Fixed b7fa5c0. Just for curiosity: why is it outsourced and what is the story# for this?

Reply:
There's no story for this I think but it is outsourced because we need a lot of functionality that was originally done in wit in other places as well (e.g. auth, tenant, notifications, ...). @xcoulon is there a story for this?